main/ui/src/main/java/org/cryptomator/ui/vaultoptions/MountOptionsController.java

```diff
@@ -52,13 +53,19 @@
 	public RadioButton mountPointCustomDir;
 	public ChoiceBox<String> driveLetterSelection;
 
+	//FUSE + Windows -> Disable some (experimental) features for the user because they are unstable
+	//Use argument Dfuse.experimental="true" to override
+	private final BooleanBinding fuseAndWindows;
```

Comment:
Please rename this variable, because right now the naming can is misleading. Suggestion: `experimentalFuseOnWindows`